real_intent/client.py

```
@@ -103,24 +110,39 @@ def __request(self, request: Request) -> dict:
+                    log(
+                        "error", 
+                        (
+                            f"Request failed after {self.max_request_attempts} "
+                            f"attempts. Error: {e}"
+                        )
+                    )
+                    raise
+
+                # Log it, sleep, loop again to try again
+                sleep_time = round(
+                    number=random.uniform(30 * n_attempt, (30 * n_attempt) + 10),
```

Ruff (0.14.5)

Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)